lib/utils/downloadTemplateFromRepo.js

```diff
@@ -152,7 +159,7 @@ function parseRepoURL(inputUrl) {
 
   switch (url.hostname) {
     case 'github.com': {
```

Comment:
With the proposal above you'd also need to update this `switch` / `case slightly. You could check the whole domain rather than just the hostname.

In your case you could check for `github.com` and `github.` which would indicate that GitHub is involved.

Reply:
Ok, so here I had 2 choices: try to make a switch case with expression in the case statement (as the switch works by checking if "switchValue" === "caseValue") to find the case with 'github.', or change to if / elseif.
The first solution was something like:
```javascript
switch(true) {
  case (url.hostname === 'github.com' || url.hostname.indexOf('github.') !== -1):
    ...
}
```
I thought that it was not super clean, so i decided to use the 2nd solution and i replaced the switch case with a simpler if / elseif chain.

Let me know what you think.
/kalote